mobile/src/Student/components/LoadingIndicator/index.js

```diff
@@ -1,4 +1,5 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState } from 'react'
+import { useFocusEffect } from '@react-navigation/native'
 import { View, StyleSheet, Text } from 'react-native'
```

Comment:
using useFocusEffect to prevent useEffects running on screens that aren't focused